### PR TITLE
Documentation - RelateObjects

### PR DESCRIPTION
**Pull request by AnneCarpenter**
I moved "Distances" to be a child measurement rather than a parent one, please confirm that's accurate.

### CODE REVIEW

**Comment by bethac07** [comment]
Haven't checked rest of PR yet but that is accurate.  Nice catch.

File: cellprofiler/modules/relateobjects.py [hunk 1]
```diff
@@ -12,13 +12,17 @@
...
-An object will be considered a child even if the edge is the only part
-touching a parent object. If an child object is touching multiple parent
+An object will be considered a child even if the edge is the only partly
+touching a parent object. If a child object is touching multiple parent
 objects, the object will be assigned to the parent with maximal overlap.
 For an alternate approach to assigning parent/child relationships,
 consider using the **MaskObjects** module.
 
-This module supports 2D and 3D objects
+If you want to include child objects that lie outside but still nearby
```
Comment (bethac07) [inline cellprofiler/modules/relateobjects.py]: nearby->near

File: cellprofiler/modules/relateobjects.py [hunk 2]
```diff
@@ -157,9 +162,9 @@ def create_settings(self):
             False,
             doc="""\
 Select "*{YES}*" to calculate the per-parent mean values of every upstream
-measurement made with the children objects and stores them as a
-measurement for the parent; the nomenclature of this new measurements is
-“Mean_<child>_<category>_<feature>”. For this reason, this module
+measurement made with the children objects and store them as a
+measurement for the parent; the nomenclature of this new measurement is
+“Mean_<child>_<category>_<feature>”. This module
 should be placed *after* all **Measure** modules that make measurements
```
Comment (bethac07) [inline cellprofiler/modules/relateobjects.py]: Could you change "should" to "must" (or even "MUST") to make it more clear that this is mandatory?  This is a confusion point for a large number of users.

File: cellprofiler/modules/relateobjects.py [hunk 3]
```diff
@@ -108,7 +113,7 @@ def create_settings(self):
             doc="""\
 Choose the method to calculate distances of each child to its parent.
```
Comment (bethac07) [inline cellprofiler/modules/relateobjects.py]: Would you be willing to add a one sentence use case? For example:

"For example, these measurements can tell you whether nuclear speckles are located more closely to the center of the nucleus or to the nuclear periphery".

**Comment by AnneCarpenter** [comment]
Great, thanks for these additions. This one is ready to go.

**Pull request merged**